python/mmSolver/utils/raytrace.py

```diff
@@ -57,7 +57,8 @@ def closest_intersect(source,
         return
 
     source_pt = OpenMaya.MFloatPoint(source[0], source[1], source[2])
-    direction_vec = OpenMaya.MFloatVector(direction[0], direction[1], direction[2])
+    direction_vec = OpenMaya.MFloatVector(direction[0], direction[1],
```

Comment:
We can replace lines like this with:
```
direction_vec = OpenMaya.MFloatVector(*direction)
```
This would reduce the line length, and would have the same result.